riscv/execute.cc

```diff
@@ -14,7 +14,7 @@ static void commit_log_reset(processor_t* p)
   p->get_state()->log_mem_write.clear();
 }
 
-static void commit_log_stash_privilege(processor_t* p)
+static void commit_stash_privilege(processor_t* p)
```

Comment:
This commit can keep the function name "commit_log_stash_privilege()."

Reply:
My bad...cut and paste error...fixed.